routes.html

```diff
@@ -45,7 +45,18 @@
         saveForm
       );
 
-      window.routeSnapper = new RouteSnapper(window.app.map, drawControls);
+      async function setupRouteSnapper() {
+        const url = `http://abstreet.s3-website.us-east-2.amazonaws.com/route-snappers/${window.app.authority}.bin`;
+        console.log(`Grabbing ${url}`);
+        const resp = await fetch(url);
+        const mapBytes = await resp.arrayBuffer();
+        window.routeSnapper = new RouteSnapper(
+          window.app.map,
+          drawControls,
+          new Uint8Array(mapBytes)
```

Comment:
The WASM bridge stuff is confusing. If I just pass the ArrayBuffer here, everything "works", but over in Rust, we see 0 bytes